client/web-sveltekit/src/lib/global-notifications/Markdown.svelte

```diff
@@ -0,0 +1,519 @@
+<script lang="ts">
+    export let dangerousInnerHTML: string = ''
+</script>
+
+{#if dangerousInnerHTML}
+    <div class='markdown'>
+        {@html dangerousInnerHTML}
+    </div>
+{/if}
```

Comment:
I've never understood why our Markdown components don't actually parse the markdown. IMO that makes more sense.

I'd do

```suggestion
<script lang="ts">
    import {renderMarkdown} from '$lib/common'
    export let content: string
</script>

{#if content}
    <div class='markdown'>
        {@html renderMarkdown(content)}
    </div>
{/if}
```

`content` is up for debate but I wouldn't call it `dangerousInnerHTML`.